scripts/workers/code_upload_submission_worker.py

```diff
@@ -3,16 +3,15 @@
 import os
 import signal
 import sys
-import yaml
 import time
 
-
-from worker_utils import EvalAI_Interface
-
+import yaml
 from kubernetes import client
-
 # TODO: Add exception in all the commands
 from kubernetes.client.rest import ApiException
+from worker_utils import EvalAI_Interface
```

Comment:
Why do you need the EvalAI_Interface here?

Reply:
I do not, I might have added it by mistake. Also will fix the formatting changes due to my version of black.